plugins/module_utils/network/eos/argspec/bgp_global/bgp_global.py

```diff
@@ -366,7 +366,10 @@ def __init__(self, **kwargs):
                             },
                         },
                         "graceful_restart_helper": {"type": "bool"},
-                        "peer": {"type": "str"},
+                        "peer": {
```

Comment:
If we do this, then both `peer` and `neighbor_address` would appear in facts. We should make the `neighbor_address` the main key and `peer` as the alias. That way we also encourage users to use `neighbor_address` and not the old one.

Reply:
Done